### PR TITLE
Pass the loss label, not loss function, as an argument

### PR DESCRIPTION
**Pull request by damkotdo**
SeqNN.evaluate() takes parameter loss and compares it to 'bce' label. However, here the loss function was passed as an argument instead of a label. This relates to #106.

### CODE REVIEW

**Comment by davek44** [comment]
Thanks for flagging this. Your resolution isn't quite right, because basenji_test.py needs to be able to pass tf keras loss function objects. I pushed an alternative fix to master.
